install_config/master_node_configuration.adoc

```diff
@@ -689,7 +689,8 @@ Each audit log contains two entries:
 .. The source IP of the request
 .. The HTTP method being invoked
 .. The original user invoking the operation
-.. The impersonated user for the operation
+.. The impersonated user for the operation (`self` meaning himself here)
```

Comment:
It's basically himself. When you're not using `--as` flag you're acting as yourself, this is when `self` will show up here, which is most of the time. When you do specify `--as` flag for any command this will be the username which you specified as the flag value. I hope that makes sense :)